lib/rules/media-feature-range-notation/__tests__/index.mjs

```diff
@@ -151,6 +151,86 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['context', { except: ['exact-value'] }],
+	fix: true,
+	computeEditInfo: true,
+
+	accept: [
+		{
+			code: '@media (width: 1px) {}',
+		},
+		{
+			code: '@media (1px: width) {}',
+		},
+		{
+			code: '@media (width >= 1px) {}',
+		},
+		{
+			code: '@media (1px <= width <= 2px) {}',
+		},
+	],
+	reject: [
+		{
+			unfixable: true,
+			code: '@media (width = 1px) {}',
```

Comment:
Should we convert it to `@media (width: 1px) {}`?

Reply:
If you are talking about adding a prefix fixer as a counterpart of the existing context filter, I think that would be out of scope for this PR.
AFAIK we have 2 rules that are affected:
- `media-feature-range-notation`
- `color-function-notation`